### PR TITLE
Fix some broken links and mapping cleanup (2.3)

### PR DESCRIPTION
**Pull request by Holzhaus**
Partial fix for #285.

### CODE REVIEW

File: source/hardware/controllers/pioneer_ddj_sx.rst [hunk 1]
```diff
@@ -496,7 +496,7 @@ As soon as loop slice mode is deactivated (by pressing [**SLICER**] mode button
 Effect Functions
 ----------------
 
-This controller mapping uses the `standard Mixxx mapping for effects sections on controllers <standard_effects_mapping>`__.
+This controller mapping uses the `standard Mixxx mapping for effects sections on controllers <https://github.com/mixxxdj/mixxx/wiki/Standard-Effects-Mapping>`__.
```
Comment (ronso0) [inline source/hardware/controllers/pioneer_ddj_sx.rst:499]: While you're at it, do you mind fixing the internal links below to #Slicer description and #Loop/Loop-roll description ?

**Comment by ronso0** [comment]
LGTM so far.
let me know when you're finished.

**Comment by ronso0** [comment]
We should not link to https://github.com/mixxxdj/mixxx/wiki/Effects-Framework
That page is marked as WIP but is oudated. I started to fix it but it's a mess currently..

File: source/hardware/controllers/pioneer_ddj_sx.rst (hunk 1 above)
Comment (Holzhaus) [reply source/hardware/controllers/pioneer_ddj_sx.rst:344]: Ended up reformatting the whole file.

**Comment by Holzhaus** [comment]
Done, there are still some broken links but I have no intention to fix them here.

**Comment by Holzhaus** [comment]
Okay, this is ready now, I promise. ;-)
Please review and merge.

**Comment by Holzhaus** [comment]
I lied. But now it's definitely ready.

**Comment by Holzhaus** [comment]
Merge?

**Comment by ronso0** [comment]
Will check this soon...

File: source/hardware/controllers/stanton_djc_4.rst [hunk 2]
```diff
@@ -56,174 +55,116 @@ The **Master** and **Mic Level** are hardware controls and interact directly wit
...
+1                         :hwlabel:`ON`                                           Toggle FX 1 for decks 1/3 (both on the left) and FX 2 for decks 2/4 (both on the right).
+2-5                       *Various*                                               See `Standard effects mapping <https://github.com/mixxxdj/mixxx/wiki/Standard-Effects-Mapping>`__.
+========================  ======================================================  ===========================================================================================
 
 Tweakables
-~~~~~~~~~~
+^^^^^^^^^^
 
-At the top of the file \`Stanton-DJC-4-scripts.js\` there are a few customizable options to change the default mapping.
+At the top of the file file:`Stanton-DJC-4-scripts.js` there are a few customizable options to change the default mapping.
```
Comment (ronso0) [inline source/hardware/controllers/stanton_djc_4.rst:167]: The wording for `autoShowFourDecks` below is strange. Can you change that to
`If a track gets loaded into deck 3 or 4, automatically show four decks in Mixxx`

File: source/hardware/controllers/stanton_djc_4.rst [hunk 3]
```diff
@@ -232,3 +173,17 @@ autoShowFourDecks If a track gets loaded into deck 3 or 4, should automatically
 showMasterVu      If set to false, show channel VU meter instead of Master L/R                               true
 dryWetAdjustValue Amount the dryWetKnob changes the value for each increment                                 0.05
 ================= ========================================================================================== =======
+
+Known Issues
+^^^^^^^^^^^^
+
+The following controls not mapped in this mapping:
```
Comment (ronso0) [inline source/hardware/controllers/stanton_djc_4.rst:180]: The following controls **are** not mapped in this mapping

Comment (Holzhaus) [reply source/hardware/controllers/stanton_djc_4.rst:180]: I'm on my phone, can you make a suggestion? Then I can commit this right now ;-)

Comment (Holzhaus) [reply source/hardware/controllers/stanton_djc_4.rst:180]: Nevermind, I'll just grab my laptop.

Comment (ronso0) [reply source/hardware/controllers/stanton_djc_4.rst:180]: for some reason I could not make a suggestion...

**Comment by ronso0** [comment]
thanks, LGTM!

**Pull request merged**